packages/runtime/src/jest-setup.ts

```diff
@@ -42,3 +42,5 @@ if (typeof window !== 'undefined') {
     })),
   })
 }
+
+;(global as any).PACKAGE_VERSION = require('../package.json').version
```

Comment:
`PACKAGE_VERSION` is never defined, but is substituted at build time by `tsup`: https://github.com/makeswift/makeswift/blob/2135f656acd1b7c19a7c195f270785bb6d41bfc6/packages/runtime/tsup.config.ts#L18

Define it as a global constant for tests.